src/components/YamlEditor.js

```diff
@@ -35,45 +35,49 @@ export default class YamlEditor extends Component {
     const { yamlData } = this.state;
     history.push(_.cloneDeep(yamlData));
     yamlData.experience[id] = data;
-    this.setState({ yamlData });
-    this.save();
+    this.setState({yamlData});
+    //this.save();
```

Comment:
Please don't leave commented out code unless it's really important. in that case add a comment describing why this code is commented out